<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifierPrenom</class>
 <widget class="QDialog" name="ModifierPrenom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>37</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>CNE :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>l'ancient prenom :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nouveau prenom :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCNE">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtAprenom">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNprenom">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveNouveauPrenom">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
